big_tests/tests/rest_client_SUITE.erl

```diff
@@ -1081,3 +1158,10 @@ break_stuff(Config) ->
 room_jid(RoomID, Config) ->
     MUCLightHost = ?config(muc_light_host, Config),
     <<RoomID/binary, "@", MUCLightHost/binary>>.
+
+set_mod_config(K, V, Config) ->
```

Comment:
Duplicate of the same function from `muc_light_SUITE` and `muc_light_legacy_SUITE`. Might be extracted from all 3 suites to `muc_light_helper`.